js-intro.js

```diff
@@ -7,25 +7,49 @@
 // --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Farhenheit.
 // Use the test variables provided below. Expected outcome: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
+
 var temp1 = 35
 var temp2 = 350
 var temp3 = 212
 
+// create a function called boilingPoint
+const boilingPoint = (number) => {
+// if number is less than 212 return 'number is below boiling point', if equals 12 return 'number is at boiling point', if number is above boiling point return 'number is above boiling point.' (if/else statement)'
+        if  (number < 212) {
+        return `${number} is below the boiling point.`
+    }
+    else if (number === 212){
+        return `${number} is at the boiling point`
+    }
+    else if (number > 212) {
+        return `${number} is above the boiling point`
+        }
+}
+console.log(boilingPoint(temp1));
+console.log(boilingPoint(temp2));
+console.log(boilingPoint(temp3));
+
 
 
 
 // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
 // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
 
 var myNumbers1 = [3, 7, 0, 6, -9]
-
+for (let i = 0; i < myNumbers1.length; i++) {
+    console.log(myNumbers1[i] * 5);
+}
 
 
 
 // --------------------3) Create a function that multiplies each number in the array by 5 using map.
 // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 
 var myNumbers2 = [8, -7, 0, 6, 2]
```

Comment:
Great job with this logic! Just make sure you wrap all of this into a reusable function.